examples/matRad_example1_phantom.m

```diff
@@ -18,7 +18,27 @@
 % (i) how to create arbitrary ct data (resolution, ct numbers)
 % (ii) how to create a cst structure containing the volume of interests of the phantom
 % (iii) generate a treatment plan for this phantom
-clc, clear, close all
+clc, close all
```

Comment:
with all examples and the matRad script you have the same preamble. I would factor this out into a separate function that can be called from the runTests m-script. In order for this to work you can get rid of the cleaning of the environment in the beginning of all scripts. and this separate function should also only be called from the runTests script so that matRad starter that only want to work with the examples do not get confused!

Reply:
If I do that though, the examples wouldn't run by themselves cause `param` won't be defined. right?

Reply:
I thought the param is always an optional thing. if nothing is passed then default values are used which are set within the function?!